package.json

```diff
@@ -216,6 +216,12 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "puppetFacts",
+        "contents": "No facts foundn\n [Refresh](command:puppet.refreshFacts)"
```

Comment:
Does this need to be more friendlier e.g. "Please hit Refresh" or does being terse perfectly fine?

Reply:
Nope.  Terse is fine.  Ignore.